appserver/node-express/lib/db-client/contributor/index.js

```diff
@@ -24,6 +27,7 @@ funcs.patchReputation = function (txid, id, repChange) {
   ).result();
 };
 
+// TODO I don't see patchVoteCount() called anywhere in /lib. Can we delete?
```

Comment:
This may be a bug, I don't recall where the spec landed, let's double-check that we aren't supposed to be calling it before we delete it.